src/datasets/features/audio.py

```diff
@@ -92,7 +94,22 @@ def encode_example(self, value: Union[str, dict]) -> dict:
             return {"bytes": buffer.getvalue(), "path": None}
         elif value.get("path") is not None and os.path.isfile(value["path"]):
             # we set "bytes": None to not duplicate the data if they're already available locally
-            return {"bytes": None, "path": value.get("path")}
+            if value["path"].endswith("pcm"):
+                # "PCM" only has raw audio bytes
+                if self.sampling_rate is None:
+                    # At least, If you want to convert "PCM-byte" to "WAV-byte", you have to know sampling rate
+                    raise KeyError("To use PCM files, please specify a 'sampling_rate' in Audio object")
+                if value.get("bytes"):
+                    # If we already had PCM-byte, we don`t have to make "read file, make bytes" (just use it!)
+                    bytes_value = np.frombuffer(value["bytes"], dtype=np.int16).astype(np.float32) / 32767
+                else:
+                    bytes_value = np.memmap(value["path"], dtype="h", mode="r").astype(np.float32) / 32767
+
+                buffer = BytesIO(bytes())
+                wavfile.write(buffer, self.sampling_rate, bytes_value)
+                return {"bytes": buffer.getvalue(), "path": value.get("path")}
```

Comment:
The path points to the pcm file, while here we are passing the new WAV data, so we need to set the path to None:
```suggestion
                return {"bytes": buffer.getvalue(), "path": None}
```

Reply:
oh! i think if we have converted pcm2wav file bytes, we can open to soundfile!
so we don`t need path good idea!